tests/hpc/slurm_master.pm

```diff
@@ -420,20 +421,49 @@ sub extended_hpc_tests ($master_ip, $slave_ip) {
     parse_extra_log('XUnit', './results/TEST-hpc-test.xml');
 }
 
+our $slurm_pkg = 'slurm';
+
 sub run ($self) {
     select_serial_terminal();
     my $nodes = get_required_var('CLUSTER_NODES');
     my $slurm_conf = get_required_var('SLURM_CONF');
     my $version = get_required_var('VERSION');
 
+    if (get_var('SLURM_VERSIONED_TEST', '0') == '1') {
+        if (is_sle('<15-SP4')) {
+            record_info 'No testing', 'versioned Slurm not used';
+            return;
+        }
+        my $query_output = script_output(qq{zypper se slurm | awk '{print \$2}' | grep -E '^slurm_[0-9]{1,2}_[0-9]{1,2}\$' | uniq});
+        my @slurm_vers = split(/(\ |\n)/, $query_output);
+        my $higher_slurm_ver = 0;
+        record_info "@slurm_vers", 'slurm packages found';
+        foreach my $v (@slurm_vers) {
+            my ($slurm_version_tmp) = $v =~ /([0-9]{1,2}_[0-9]{1,2})/;
+            $slurm_version_tmp =~ s/_/\./;
+            if ($higher_slurm_ver < $slurm_version_tmp) {
+                $higher_slurm_ver = $slurm_version_tmp;
+            }
+        }
+        $higher_slurm_ver =~ s/\./_/;    # restore format
+        record_info "Latest $higher_slurm_ver", "This should be the latest/higher slurm version";
+        $slurm_pkg = 'slurm_' . $higher_slurm_ver;
+    }
+
     barrier_wait('CLUSTER_PROVISIONED');
     $self->prepare_user_and_group();
     $self->generate_and_distribute_ssh();
 
     # provision HPC cluster, so the proper rpms are installed,
     # munge key is distributed to all nodes, so is slurm.conf
     # and proper services are enabled and started
-    zypper_call('in slurm slurm-munge slurm-torque');
+    if ($slurm_pkg =~ /slurm_/) {
+        # $slurm_pkg-munge is installed explicitly since slurm_23_02
+        # bsc1214094
+        zypper_call("in $slurm_pkg $slurm_pkg-munge");
```

Comment:
what about `$slurm_pkg-torque`? i see it bellow in non-versioned

Reply:
i dont know about this package. Maybe @schlad has an idea as he was the author in first place. I cant really remember but i think `$slurm_pkg-torque` is not available.

Reply:
Ok, lets wait for @schlad

Reply:
just to summarize; we talked and agreed this thing is very much a product related thing, so @b10n1k is the one who can really check/answer this.
This is a simple matter of slurm-torque and answering if this is an rpm which is versioned/non-versioned and checking if it does install nicely in case slurm is in versioned flavor?

I proposed we could verify that manually together but we would need a cluster with provisioned slurm scheduled. 
Surely we could just search with zypper and see how that package looks like in case the slurm is versioned.

Reply:
i think this can be resolved